src/TestFramework/TestFramework/Attributes/TestMethod/OSConditionAttribute.cs

```diff
@@ -32,7 +32,9 @@ public OSConditionAttribute(ConditionMode mode, OperatingSystems operatingSystem
         : base(mode)
     {
         _operatingSystems = operatingSystems;
-        IgnoreMessage = $"Test is only supported on {operatingSystems}";
+        IgnoreMessage = mode == ConditionMode.Include
```

Comment:
I am wondering if we should do some split/join to have something like `Test is only supported on the following OSes: 'Windows', 'MacOS'`

Reply:
In case of `[OSCondition(ConditionMode.Exclude, OperatingSystems.Windows)]`, I find it better to read as:

```
Test is not supported on Windows
```

rather than

```
Test is only supported on the following OSes: 'Linux', 'OSX', 'FreeBSD'
```

Reply:
Sorry looks like I wasn't clear :) I like to have 2 messages, I am just wondering what's being display when you bitwise the enum.

Reply:
`ToString` will already comma-separate the named values:

![image](https://github.com/user-attachments/assets/0c7bedcb-5622-4e74-8117-931c078fd356)

Reply:
Oh cool, I wasn't aware of it. Approving then.